src/components/PuzzleGameContainer/index.jsx

```diff
@@ -1,37 +1,46 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 import { Col, Row, Card } from "antd";
 import habitatImg from "./../../images/puzzleGame/habitat.jpg";
-import yellowSpotImg from "./../../images/yellow-spot.svg";
+import conservationImg from "./../../images/puzzleGame/conservation.jpg";
+import holdingPlantImg from "./../../images/puzzleGame/holdingPlant.jpg";
+import plantingImg from "./../../images/puzzleGame/planting.jpg";
+import PuzzleGrid from "./../PuzzleGrid";
 import "./index.css";
 
 // This component contains the list of images the user can choose to solve
 // it also renders the PuzzleGrid component
 // When the user clicks on an image it will be passed to the PuzzleGrid component as a prop
 // PuzzleGrid then renders the puzzle tiles of that image
 const PuzzleGameContainer = () => {
+  const images = [habitatImg, conservationImg, holdingPlantImg, plantingImg];
```

Comment:
since these are constants, after imports outside of component, call array PUZZLE_IMAGE_POOL

Reply:
PTAL